### ISSUE TITLE
Model Load changes

### ISSUE THREAD

**Issue by Edgarware** [jmaliksi/blaseball-mike#119]
We have a lot of model objects with a lot of different ways to load them, which gets confusing for new users and complicates implementation and testing. The current load split has mostly been to differentiate between the different sources of data (blaseball vs chronicler vs datablase) but as we add more endpoints (especially now that chronicler v2 exposes historical data for *everything*) this will get unwieldy. I also think most end-users just want to say "I want this/these objects at this time or day" and are less concerned about where the data comes from. There's also some inconstancy in the current loads, like Team.load only returning one team and Tiebreaker.load only supporting one ID but still returning a dictionary.

The proposed change would require as many models as possible to have the following functions:

* `Object.load_one(id, season=None, day=None, time=None)`
   * returns a single model given available filters
   * replaces load_one and load_one_at_time
* Object.load(ids, season=None, day=None, time=None)
   * returns a dictionary of multiple objects given available filters
   * replaces load, load_by_season, load_by_
* Object.load_history(id)
   * returns a generator of all changes to this object over time
   * same as player/team load_history but extended to all types

**Comment by jmaliksi** [comment]
Yeah these sound great. Might even define those as abstract methods on the
base model.
